docs/data/joy/getting-started/tutorial/tutorial.md

```diff
@@ -0,0 +1,232 @@
+
+```jsx
+// ...other imports
+import TextField from '@mui/joy/TextField';
+
+<Sheet
+  variant="outlined"
+  sx={
+    {
+      // ...
+    }
+  }
+>
+  ...typography
+  <TextField
+    // html input attribute
+    name="email"
+    type="email"
+    placeholder="johndoe@email.com"
+    // pass down to FormLabel as children
+    label="Email"
+  />
+  <TextField
+    name="password"
+    type="password"
+    placeholder="password"
+    label="Password"
+  />
+</Sheet>;
+```
+
+### 5. Use `Button` and `Link` for actions
+
+We will render a log in button and a sign up link to navigate to a different page.
+
+- The default `Button` comes with the `contained` variant and `primary` color.
+- Another typography is used to add the text which is aligned at the center via the `sx` prop.
+- The `Link` is rendered at the end of the typography.
+
+```jsx
+// ...other imports
+import Button from '@mui/joy/Button';
+import Link from '@mui/joy/Link';
+
+<Sheet
+  variant="outlined"
+  sx={
+    {
+      // ...
+    }
+  }
+>
+  ...typography and text-fields
+  <Button
+    sx={{
+      mt: 1, // margin top
+    }}
+  >
+    Log in
+  </Button>
+  <Typography
+    endDecorator={<Link href="/sign-up">Sign up</Link>}
+    fontSize="sm"
+    sx={{ alignSelf: 'center' }}
+  >
+    Don't have an account?
+  </Typography>
+</Sheet>;
+```
+
+Try changing the button's variant between `text`, `outlined`, `light` or `contained`, and goes with the one that you like!.
+
+<!-- TODO: Add the result image -->
+
+## 🎁 Bonus: Dark mode
+
+Joy provides an effortless way to toggle between modes by using the react hook `useColorScheme`. All you need to do is creating a component that use the hook and then renders it under the `CssVarsProvider`.
+
+```jsx
+import React from 'react';
+import { CssVarsProvider, useColorScheme } from '@mui/joy/styles';
+// ...other imports
+
+const ModeToggle = () => {
+  const { mode, setMode } = useColorScheme();
+  const [mounted, setMounted] = React.useState(false);
+
+  // necessary for server-side rendering
+  // because mode is undefined on the server
+  React.useEffect(() => {
+    setMounted(true);
+  }, []);
+  if (!mounted) {
+    return null;
+  }
+
+  return (
+    <Button
+      variant="outlined"
+      onClick={() => {
+        if (mode === 'light') {
+          setMode('dark');
+        } else {
+          setMode('light');
+        }
+      }}
+    >
+      {mode === 'light' ? 'Turn dark' : 'Turn light'}
+    </Button>
+  );
+};
+
+export default function App() {
+  return (
+    <CssVarsProvider>
+      <ModeToggle />
+      <Sheet>...</Sheet>
+    </CssVarsProvider>
+  );
+}
+```
+
+> 💡 By using `useColorScheme` hook, Joy ensures that the user selected mode (stored in localStorage by default) is in sync across the browser tabs.
+
+Congratulations 🎉! you have completed the turorial.
+
+## What's next?
+
+- 💎 Check out all the features of Joy
+- 💅 Learn more about customization
```

Comment:
```suggestion
- 💅 Learn about how to customize Joy components
```

Reply:
For this one, my purpose is to link to the customization overall page not specific to components.

Reply:
But what would you customize if not components? 😅

Reply:
Apart from components are theme tokens (font, typography, palette, shadow, ...etc). Should we merge them into component customization?

Reply:
Yeah—in a strict sense, these are not components. But you would be customizing them as a way to change the component's look, so it's a matter of which strategy you pick to do it. I'd keep the "Learn how to customize Joy components", we could as a hook to "using the theme", "using sx prop", etc.